platform/security/docs/src/main/resources/ManContents.adoc

```diff
@@ -109,3 +109,120 @@ The following packages have been exported by the DDF Security application and ar
 * Security PEP
 * Security PDP
 * Security STS
+
+=== IdP/SP
+
+The {branding} Security IdP application provides service provider handling that satisfies the http://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf[SAML 2.0 Web SSO profile] in order to support external IdPs (Identity Providers).
+
+==== Installation
+
+The IdP bundles are not installed by default.
+They can be started by installing the `security-idp` feature.
+
+==== Security IdP Service Provider
+
+The IdP client that interacts with the specified Identity Provider.
+
+===== Configuration
+
+. Navigate to Admin Console -> DDF Security -> Configuration -> IdP Client
+. Populate IdP Metadata field through one of the following:
+.. An HTTPS URL (https://)
+.. A file URL (file:)
+.. An XML block to refer to desired metadata (e.g., https://localhost:8993/services/idp/login/metadata)
+
+.IdP Client (SP) example.xml
+[source,xml,linenums]
+----
+<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" entityID="https://localhost:8993/services/saml">
+  <md:SPSSODescriptor protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
```

Comment:
It looks like we have the wrong metadata for each section. The IdP Client should be getting the IDPSSODescriptor metadata and the IdP Server should be getting the SPSSODescriptor metadata.